pkg/master/master.go

```diff
@@ -316,7 +319,8 @@ func (m *Master) InstallAPIs(c *Config) {
 		apiGroupInfo := genericapiserver.APIGroupInfo{
 			GroupMeta: *batchGroupMeta,
 			VersionedResourcesStorageMap: map[string]map[string]rest.Storage{
-				"v1": batchResources,
+				"v1":       batchv1Resources,
+				"v2alpha1": batchv2alpha1Resources,
```

Comment:
I think you should probably check that the version is enabled before adding to this map, so like construct the map here and have two `if c.APIResourceConfigSource.AnyResourcesForVersionEnabled(batchapiv2alpha1.SchemeGroupVersion) { apiGroupInfo.VersionedResourcesStorageMap["v2alpha1"] = batchv2alpha1Resources }` statements afterwards.

I don't think it's that big of a deal but without this there's no way to e.g. turn the alpha version off and leave the other version on.